src/mbgl/style/layers/layer_properties.hpp.ejs

```diff
@@ -37,7 +37,13 @@ public:
 <% if (/-pattern$/.test(property.name) || property.name === 'line-dasharray') { -%>
     PaintProperty<<%- propertyType(property) %>, CrossFadedPropertyEvaluator> <%- camelizeWithLeadingLowercase(property.name) %> { <%- defaultValue(property) %> };
 <% } else if (property.name === 'fill-outline-color') { -%>
-    PaintProperty<<%- propertyType(property) %>> <%- camelizeWithLeadingLowercase(property.name) %> { { 0, 0, 0, -1 } };
+    PaintProperty<<%- propertyType(property) %>> <%- camelizeWithLeadingLowercase(property.name) %> { {} };
```

Comment:
Given https://github.com/mapbox/mapbox-gl-native/blob/c670b08ecb431b1bfd02da748de3d9fa16a757ec/src/mbgl/renderer/painter_fill.cpp#L27-L27, this should expand to `PaintProperty<Color> fillOutlineColor { };` so it's in the "undefined" state, not a fully-transparent color.